File: Java/FizzBuzz1.java
public class FizzBuzz1 {
    static final int LIMIT = 100;

    public static void main(String[] args) {
        for (int i = 1; i <= LIMIT; i++) {
            if (i % 15 == 0) {
                System.out.print("FizzBuzz");
            } else if (i % 3 == 0) {
                System.out.print("Fizz");
            } else if (i % 5 == 0) {
                System.out.print("Buzz");
            } else {
                System.out.print(i);
            }
            if (i < LIMIT) {
                System.out.print(", ");
            }
        }
        System.out.println();
    }
}

File: Erlang/fizzbuzz1.erl
-module(fizzbuzz1).
-export([main/0, main/1]).

-define(LIMIT, 100).

get_str(X) ->
    if X rem 15 == 0 -> "FizzBuzz";
       X rem 3 == 0 -> "Fizz";
       X rem 5 == 0 -> "Buzz";
       true -> integer_to_list(X)
    end.

main() ->
    io:fwrite("~s~n", [lists:flatten(lists:join(", ", [get_str(X) || X <- lists:seq(1, ?LIMIT)]))]).

main(_) -> main().
